README.md

```diff
@@ -226,3 +226,17 @@ import { like } from '../my-rsql-operators/like';
 
 const op = like('John Travolta'); // '=like="John Travolta"'
 ```
+
+If the value can contain special characters, `escapeValue` function can be used to escape them:
+
+```js
+new Operation(escapeValue(value), '=like=');
+```
+
+A custom list operator could look like this:
+
+```js
```

Comment:
In case you encounters any issues while linting it, I guess, I've found the fix - you need to remove `overrides.parserOptions.project` property from `.eslintrc` file

Reply:
Nope, the syntax is wrong, it's not `ts` but `typescript` - fixed.